src/rules/dynamic-import-chunkname.js

```diff
@@ -36,8 +37,10 @@ module.exports = {
 
     const paddedCommentRegex = /^ (\S[\s\S]+\S) $/;
     const commentStyleRegex = /^( ((webpackChunkName: .+)|((webpackPrefetch|webpackPreload): (true|false|-?[0-9]+))|(webpackIgnore: (true|false))|((webpackInclude|webpackExclude): \/.*\/)|(webpackMode: ["'](lazy|lazy-once|eager|weak)["'])|(webpackExports: (['"]\w+['"]|\[(['"]\w+['"], *)+(['"]\w+['"]*)\]))),?)+ $/;
-    const chunkSubstrFormat = ` webpackChunkName: ["']${webpackChunknameFormat}["'],? `;
+    const chunkSubstrFormat = `webpackChunkName: ["']${webpackChunknameFormat}["'],? `;
```

Comment:
how come this one no longer has a leading space, but still has a trailing space?

Reply:
I'm using this regex in a `replace()` operation to remove it, and I didn't want the preceding space to be removed in a comma-separated list of webpack options, it could lead to the format being incorrect after applying the suggestion (i.e there could be no space after comma). The existence of the space is already checked by the lint rule using the regex right above this.